SourceKitStressTester/Tests/StressTesterToolTests/StressTesterToolTests.swift

```diff
@@ -42,7 +43,8 @@ class StressTesterToolTests: XCTestCase {
       XCTAssertEqual(defaults.dryRun, false)
       XCTAssertEqual(defaults.reportResponses, false)
       XCTAssertEqual(defaults.conformingMethodsTypeList, ["s:SQ", "s:SH"])
-      XCTAssertEqual(defaults.swiftc, swiftcPath)
+      // FIXME: Change to swiftcPath when validate is back to normal
+      XCTAssertEqual(defaults.swiftc, nil)
```

Comment:
Does this change have any impact on the integration test?

Reply:
No, it's still correct when actually run - `customValidate` is called from `run` in that case. These tests don't call the `run` method though, so it isn't set. They do call `validate`, but the mutation isn't kept in 0.3.0 (or 0.3.1)

Reply:
Unless by "this change" you actually meant the whole change, in which case yes - it fixes the integration test (hopefully). The integration test was failing because the members set in `validate` were assumed to be set and thus were force unwrapped, which was throwing a runtime error.